dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -210,6 +210,13 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
+# Install GNMI client
+RUN mkdir -p /tmp/gnmi \
+    && cd /tmp/gnmi \
+    && wget "https://sonic-build.azurewebsites.net/api/sonic/artifacts?branchName=master&definitionId=142&artifactName=sonic-buildimage.vs&target=target/debs/bullseye/sonic-gnmi_0.1_amd64.deb" -O sonic-gnmi_0.1_amd64.deb \
```

Comment:
>sonic-gnmi_0.1_amd64.deb [](http://example.com/codeflow?start=196&length=24)

docker-ptf is using [https://github.com/lguohan/gnxi.git](https://github.com/lguohan/gnxi.git).
Please check if you can reuse that client.
We intend to reduce using sonic binary in the test controller.